fault.h

```diff
@@ -24,6 +24,20 @@
  */
 #define KIBOSH_FAULT_TYPE_STR_LEN 32
 
+/**
+ * Constant flags for byte corruption modes.
+ */
+#define CORRUPT_ZERO 1000
+#define CORRUPT_RAND 1001
+#define CORRUPT_ZERO_SEQ 1100
+#define CORRUPT_RAND_SEQ 1101
+#define CORRUPT_DROP 1200       // this will silently drop data from write calls
+
+/**
+ * Generate a random double between 0 and 1.0
+ */
+#define RAND_FRAC ((double)rand()/(double)RAND_MAX)
```

Comment:
Can we turn this into a function instead?

Reply:
Added a random_fraction function in util to replace RAND_FRAC